install.md

```diff
@@ -149,6 +149,12 @@ sudo apt-get install \
   uidmap
 ```
 
+On openSUSE Leap:
+
+```bash
+sudo zypper -n in libseccomp-devel libgpgme-devel
+```
+
```

Comment:
A note for the future.  This is a nice change and I appreciate you doing it, but I'd rather have it in a separate PR as it's unrelate to the podman info changes.  No worries for this PR though

Reply:
Fixed! :)